deeptools/heatmapper_utilities.py

```diff
@@ -15,9 +15,9 @@ def plot_single(ax, ma, average_type, color, label, plot_type='simple'):
         ax.fill_between(x, sumry, facecolor=color, alpha=0.6)
 
     elif plot_type == 'se':  # standard error
-        std = np.std(ma, axis=0)/np.sqrt(ma.shape[0])
+        std = np.std(ma, axis=0) / np.sqrt(ma.shape[0])
         alpha = 0.2
-        if type(color) == type((0, 0)):  # check if color is tuple
+        if isinstance(color, type((0, 0))):  # check if color is tuple
             # add the alpha channed to the color tuple
             f_color = [c for c in color[:-1]] + [alpha]
```

Comment:
Here there could be a potential problem because the check `if isinstance(color, type((0, 0))):` would return true regardless of the length of `color`. However, whenever the color is a tuple I suppose it is always a proper color tuple with three elements. In other words, the test, I think, wants to check if `color` is tuple or `str`. I will check this